flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/avro/AvroParquetReaders.java

```diff
@@ -51,15 +51,7 @@ public class AvroParquetReaders {
     public static <T extends SpecificRecordBase> StreamFormat<T> forSpecificRecord(
             final Class<T> typeClass) {
         return new AvroParquetRecordFormat<>(
-                new AvroTypeInfo<>(typeClass),
```

Comment:
It seems that you are reverting the changes made in #19956. Are there any specify reasons?

Reply:
Yeah I find that the change is only needed for the GenericRecord we provided in Python.